README.md

```diff
@@ -287,7 +300,7 @@ The currently supported template variables are:
 
 # Searching
 
-Joplin implements the SQLite Full Text Search (FTS4) extension. It means the content of all the notes is indexed in real time and search queries return results very fast. Both [Simple FTS Queries](https://www.sqlite.org/fts3.html#simple_fts_queries) and [Full-Text Index Queries](https://www.sqlite.org/fts3.html#full_text_index_queries) are supported. See below for the list of supported queries:
+Joplin implements the SQLite Full Text Search (FTS4) extension. It means the content of all the notes are indexed in real time and search queries return results very fast. Both [Simple FTS Queries](https://www.sqlite.org/fts3.html#simple_fts_queries) and [Full-Text Index Queries](https://www.sqlite.org/fts3.html#full_text_index_queries) are supported. See below for the list of supported queries:
```

Comment:
are vs is - fixed for grammar issue.

Reply:
Isn't "content" singular and in that case doesn't it mean it should be "is"?

Reply:
'notes' is plural, so 'is' should be 'are'. Right? :)